src/main/java/io/fabric8/vertx/maven/plugin/mojos/AbstractRunMojo.java

```diff
@@ -423,14 +422,22 @@ private boolean isYaml(String configFile) {
 
         @Override
         public void call(String kind, Path path) {
```

Comment:
Maybe `kind` should be enum ? Well, its not used anyway, so maybe it could be removed, too ?